Proteomics/Modifications/ModificationGeneral.cs

```diff
@@ -48,28 +62,33 @@ public ModificationGeneral(string _Id, string _Accession, string _ModificationTy
             this.FileOrigin = _FileOrigin;
         }
 
-        public static ModLocationOnPeptideOrProtein? Localization(string myLocalization)
+        public static string ModLocationOnPeptideOrProtein(string modLocation)
         {
-            switch (myLocalization)
+            List<string> locationList = new List<string>
             {
-                case "N-terminal.":
-                    return ModLocationOnPeptideOrProtein.NProt;
-
-                case "C-terminal.":
-                    return ModLocationOnPeptideOrProtein.ProtC;
-
-                case "Peptide N-terminal.":
-                    return ModLocationOnPeptideOrProtein.NPep;
-
-                case "Peptide C-terminal.":
-                    return ModLocationOnPeptideOrProtein.PepC;
-
-                case "Anywhere.":
-                    return ModLocationOnPeptideOrProtein.Any;
+                "N-terminal.",
+                "C-terminal.",
+                "Peptide N-terminal.",
+                "Peptide C-terminal.",
+                "Anywhere."
+            };
+            if (locationList.Contains(modLocation))
+            {
+                return modLocation;
+            }
+            else
+            {
+                return "Unassigned.";
+            }
+        }
 
-                default:
-                    return null;
+        public double? RoundedDouble(double? myNumber)
```

Comment:
this method may be more appropriate for a more global class if you plan to use this type of rounding elsewhere. up to you

Reply:
you're right. can you suggest a place?

Reply:
mzLib/Chemistry/ClassExtensions.cs

Reply:
then you can call it like:
double myNumber = 1000.29398730491820981;
double myNumberRounded = myNumber.RoundedDouble();

Reply:
should we default a number of digits or pass it 
double myNumberRounded = myNumber.RoundedDouble(9)